src/Components/SysTable/SysTable.js

```diff
@@ -1,183 +1,173 @@
 import { useIntl } from 'react-intl';
-import { useQuery } from '@apollo/client';
-import { useReactiveVar } from '@apollo/client';
-
-const initialState = {
-    tableVars: {
-        limit: 10,
-        offset: 0,
-        orderBy: 'LAST_SCAN_DATE_DESC',
-        name: ''
-    },
-    sortBy: {
-        index: 4,
-        direction: SortByDirection.desc
-    },
-    rows: []
-};
-const sortIndices = { 0: 'DISPLAY_NAME', 1: 'HAS_MATCH', 2: 'LAST_MATCH_DATE', 3: 'TOTAL_MATCHES', 4: 'LAST_SCAN_DATE' };
-const orderBy = ({ index, direction }) => `${sortIndices[index]}_${direction === SortByDirection.asc ? 'ASC' : 'DESC'}`;
-
-const tableReducer = (state, action) => {
-    switch (action.type) {
-        case 'setTableVars':
-            return { ...state, tableVars: { ...state.tableVars, ...action.payload } };
-        case 'setSortBy':
-            return { ...state, sortBy: action.payload, tableVars: { ...state.tableVars, ...action.tableVars } };
-        case 'setRows':
-            return { ...state, rows: action.payload };
-    }
-
-    return state;
-};
+import { RegistryContext } from '../../store';
 
 const SysTable = () => {
+    const inventory = useRef(null);
     const intl = useIntl();
-    const [{ tableVars, sortBy, rows }, stateSet] = useReducer(tableReducer, {
-        ...initialState
-    });
-    const { data: sysTableData, loading: sysTableLoading, error: sysTableError } =
-        useQuery(GET_SYSTEM_TABLE, { variables: { ...tableVars, ...useReactiveVar(sysTableFilters) } });
+    const { getRegistry } = useContext(RegistryContext);
+
     const columns = [
-        { title: intl.formatMessage(messages.name), transforms: [cellWidth(45), sortable] },
-        { title: intl.formatMessage(messages.lastStatus), transforms: [sortable, cellWidth(10)] },
-        { title: intl.formatMessage(messages.matched), transforms: [sortable, cellWidth(10)] },
-        { title: <span>{totalMatchesTitle({ tooltip: intl.formatMessage(messages.totalMatchesNote),
-            title: intl.formatMessage(messages.totalMatches) })}</span>, transforms: [sortable, cellWidth(15)] },
-        { title: intl.formatMessage(messages.lastScan), transforms: [sortable, cellWidth(10)] }
+        {
+            title: intl.formatMessage(messages.name),
+            transforms: [cellWidth(45), sortable],
+            key: 'displayName',
+            sortBy: ['DISPLAY_NAME'],
+            renderFunc: (displayName) => <Link to={`/systems/${displayName}`}>{displayName}</Link>
+        },
+        // {
+        //     title: 'Tags',
+        //     key: 'tags',
+        //     props: { width: 10, isStatic: true }
+        // },
+        {
+            title: intl.formatMessage(messages.lastStatus),
+            transforms: [sortable, cellWidth(10)],
+            key: 'hasMatch',
+            sortBy: ['HAS_MATCH'],
+            renderFunc: (hasMatch, id, row) => <StatusLabel isDisabled={row.isDisabled} hasMatch={hasMatch} displayMatch />
```

Comment:
We can destruct the row in here since we don't need the entrie row data.
```suggestion
            renderFunc: (hasMatch, id, { isDisabled }) => <StatusLabel isDisabled={isDisabled} hasMatch={hasMatch} displayMatch />
```